torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -37,6 +37,10 @@ inline Tensor leaky_relu(const Tensor& input, const LeakyReLUOptions& options) {
   }
 }
 
+inline Tensor log_sigmoid(const Tensor& input) {
```

Comment:
I think the Python functional is called `logsigmoid`, we probably should name it the same here.